test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -666,6 +678,34 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         $result.Error | Should BeNullOrEmpty
     }
 
+    It "Validate Invoke-RestMethod StandardMethod and CustomMethod parameter sets" {
+
+        $errorId = "AmbiguousParameterSet,Microsoft.PowerShell.Commands.InvokeRestMethodCommand"
+        { Invoke-RestMethod -Uri 'http://http.lee.io/method' -Method GET -CustomMethod TEST } | ShouldBeErrorId $errorId
+    }
+
+    It "Validate CustomMethod method is used" {
+        
+        $command = "Invoke-RestMethod -Uri 'http://http.lee.io/method' -CustomMethod TEST"
+        $result = ExecuteWebCommand -command $command
+        $result.Error | Should BeNullOrEmpty
+        $result.Output.output.method | Should Be "TEST"
+    }
+
+    It "Validate Invoke-RestMethod default ContentType for CustomMethod POST" {
+        
+        $command = "Invoke-RestMethod -Uri 'http://httpbin.org/post' -CustomMethod POST -Body 'testparam=testvalue'"
+        $result = ExecuteWebCommand -command $command
+        $result.Output.headers.'Content-Type' | Should Be "application/x-www-form-urlencoded"
```

Comment:
since you're sending a specific body, seems like you should just check it was returned correctly in the response:

$result.Output.form.testvalue | Should Be "testvalue"